tidy3d/components/structure.py

```diff
@@ -77,6 +77,21 @@ class Structure(AbstractStructure):
         discriminator=TYPE_TAG_STR,
     )
 
+    def eps_diagonal(self, frequency: float) -> Tuple[complex, complex, complex]:
```

Comment:
As discussed, I'll extend it to include `Coords`  for custom medium?

Reply:
yea, sounds good.